MediaBrowser.Providers/Music/AlbumMetadataService.cs

```diff
@@ -61,8 +61,10 @@ protected override ItemUpdateType UpdateMetadataFromChildren(MusicAlbum item, IL
 
                 var songs = children.Cast<Audio>().ToArray();
 
-                updateType |= SetAlbumArtistFromSongs(item, songs);
                 updateType |= SetArtistsFromSongs(item, songs);
+                updateType |= SetAlbumArtistFromSongs(item, songs);
```

Comment:
I would change the function definition to the following to avoid warnings.
```c#
private ItemUpdateType SetAlbumArtistFromSongs(MusicAlbum item, IReadOnlyList<Audio> songs)
```

(can do the same for the others as well)